ia/src/protocole.py

```
@@ -1,6 +1,8 @@
 import socket
 import threading
 import game
+import ml_agent
+import random
```

Pylint (3.3.7)

[convention] standard import "random" should be placed before third party imports "game", "ml_agent"

(C0411)